examples/cluster-with-static.conf.example

```diff
@@ -0,0 +1,27 @@
+##--------------------------------------------------------------------
+## Cluster in service discovery via static nodes mode
+##
+## Configs to instruct how individual nodes can discover each other
+##--------------------------------------------------------------------
+## Note: This is an example of how to configure this feature
+##       you should copy and paste the below data into the emqx.conf for working
+
+cluster {
+    ## Human-friendly name of the EMQX cluster.
+    name = emqxcl
+
+    ## Service discovery method for the cluster nodes
+    discovery_strategy = static
+
+    ## List of core nodes that the replicant will connect to
+    core_nodes = ['emqx1@192.168.0.1', 'emqx2@192.168.0.2']
+
+    ## Remove disconnected nodes from the cluster after this interval
+    autoclean = 5m
+
+    ## If true, the node will try to heal network partitions automatically
+    autoheal = true
+
+    ## List EMQX node names in the static cluster
+    static.seeds = ['emqx1@192.168.0.1', 'emqx2@192.168.0.2']
```

Comment:
all `['emqx1@192.168.0.1', 'emqx2@192.168.0.2']` is bad  hocon format.
should be 
```
"emqx1@192.168.0.1,emqx2@192.168.0.2"
% or
["emqx1@192.168.0.1", "emqx2@192.168.0.2"]
```
don't support `'` in hocon.